src/app/containers/CpsAssetPageMain/index.jsx

```diff
@@ -22,6 +25,7 @@ const CpsAssetPageMain = ({ pageData }) => {
       <LinkedData type="Article" seoTitle={title} />
       <Grid as="main" role="main">
         <GridItemConstrainedMedium>
+          <Link to="/pidgin/test-12345678">Test MAP to MAP inline link</Link>
```

Comment:
Add a puppeteer test for the onward journey for this `link` to ensure the `rich-text-transforms` bundle is loaded

Reply:
Added by @dr3 🚀